### PR TITLE
Add traditional vs serverless databases article

### PR DESCRIPTION
**Pull request by imchairmanm**

### CODE REVIEW

File: content/11-serverless/04-traditional-vs-serverless-databases.mdx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+While these assets have traditionally been operated by the organization itself, their management has increasingly been outsourced to external providers.  The first iteration of this idea was to move to *managed databases*, where a provider manages the infrastructure and software while allowing the user to adjust settings and set scaling policies.
+
+More recently, this idea has evolved to *serverless databases*, where the entire infrastructure, data storage, and underlying architecture is entirely managed by the provider and the user only accesses the data they manage through API-like interactions.
+
+In this guide, we'll talk about the differences between traditionally deployed databases and their serverless counterparts.  We will discussed managed database offerings where appropriate to help distinguish between the different options providers offer.  Learning about these different concepts can help you understand the trade-offs between different offerings and make an informed decision about what type of database management would make sense for your own projects.
```
Comment (alex-emerich) [inline content/11-serverless/04-traditional-vs-serverless-databases.mdx:15]:
```suggestion
In this guide, we'll talk about the differences between traditionally deployed databases and their serverless counterparts.  We will discuss managed database offerings where appropriate to help distinguish between the different options providers offer.  Learning about these different concepts can help you understand the trade-offs between different offerings and make an informed decision about what type of database management would make sense for your own projects.
```

File: content/11-serverless/04-traditional-vs-serverless-databases.mdx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+## Conclusion
+
+In this article, we took a look at some of the key differences between traditional, self-managed databases and serverless database services.  We talked about how they are different from a user's perspective and what trade-offs you must consider when deciding on a database strategy depending on your organization's priorities.
+
+While serverless databases are not suitable for every type of application, they can make database management and operation significantly easier for many organizations.  Understanding when the might be a good solution and what shortcomings they might have can help you figure out if they can and should be considered when evaluating database solutions.
```
Comment (alex-emerich) [inline content/11-serverless/04-traditional-vs-serverless-databases.mdx:126]:
```suggestion
While serverless databases are not suitable for every type of application, they can make database management and operation significantly easier for many organizations.  Understanding when they might be a good solution and what shortcomings they might have can help you figure out if they can and should be considered when evaluating database solutions.
```

Comment (alex-emerich) [inline content/11-serverless/04-traditional-vs-serverless-databases.mdx:126]: Might be good to outlink to the prisma data platform at the end for serverless users or those considering serverless.

**Review by alex-emerich — APPROVED** [review]
Looks really good!

File: content/11-serverless/04-traditional-vs-serverless-databases.mdx (hunk 2 above)
Comment (imchairmanm) [reply content/11-serverless/04-traditional-vs-serverless-databases.mdx:126]: Great idea. Thanks for the reminder on that one.

**Pull request merged**